packages/docusaurus-plugin-openapi/src/sidebars/index.ts

```diff
@@ -42,6 +38,8 @@ type ApiItem = Pick<ApiPageMetadata, keys> & {
 
 type Item = InfoItem | ApiItem;
 
+const CategoryMetadataFilenameBase = "_category_";
```

Comment:
no way to extract this from the docusaurus lib?

Reply:
Yea they don’t export it anymore, I’m fine with what you have here